packages/edit-site/src/components/app/index.js

```diff
@@ -30,9 +31,14 @@ function AppLayout() {
 
 export default function App() {
 	useRegisterSiteEditorRoutes();
-	const routes = useSelect( ( select ) => {
-		return unlock( select( editSiteStore ) ).getRoutes();
+	const { routes, currentTheme, editorSettings } = useSelect( ( select ) => {
+		return {
+			routes: unlock( select( editSiteStore ) ).getRoutes(),
+			currentTheme: select( coreStore ).getCurrentTheme(),
+			editorSettings: select( editSiteStore ).getSettings(),
```

Comment:
Let's note that this is a temp solution until the `has_theme_json` value is available for the current theme.